packages/terra-demographics-banner/src/selectable-demographics-banner/SelectableDemographicsBanner.jsx

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl, intlShape } from 'react-intl';
+import classNames from 'classnames/bind';
+import * as KeyCode from 'keycode-js';
```

Comment:
can we change the keycode imports to only pull in the codes we need?

Reply:
updated : [here](https://github.com/cerner/terra-core/blob/d063a9d511619a4181c537d2030c86f777acf38b/packages/terra-demographics-banner/src/selectable-demographics-banner/SelectableDemographicsBanner.jsx#L5)